src/DropArea/index.ts

```diff
@@ -0,0 +1,85 @@
+import {Component, cloneElement} from 'react';
+import {render} from 'react-universal-interface';
+import {h, noop} from '../util';
+
+export interface IDropAreaProps {
+  onFiles?: (files: File[], event?) => void;
+  onText?: (text: string, event?) => void;
+  onUri?: (url: string, event?) => void;
+}
+
+export interface IDropAreaState {
+}
+
+export class DropArea extends Component<IDropAreaProps, IDropAreaState> {
+  static defaultProps = {
+    onFiles: noop,
+    onText: noop,
+    onUrl: noop,
+  };
+
+  mounted = true;
+
+  coponentWillUnmount () {
+    this.mounted = false;
+  }
+
+  onDragOver = (originalDragOver) => (event) => {
+    (originalDragOver || noop)(event);
```

Comment:
why don't you make `noop` the default value of `originalDragOver` param?

Reply:
Nice idea, will do.